rpc_jobs/phobos.yml

```diff
@@ -192,3 +198,4 @@
           """
         }
       } // phobos node
+      } // timestamps
```

Comment:
Can you indent the use_node block?

Reply:
I @mattt416 I added the indent - it seems to have thrown off the formatting when viewing the commit diff. I'm pretty sure it's all the same apart from the timestamp wrapper, but just double check for me...